src/cls/rgw/cls_rgw.cc

```diff
@@ -3078,9 +3074,26 @@ int rgw_user_usage_log_trim(cls_method_context_t hctx, bufferlist *in, bufferlis
   if (ret < 0)
     return ret;
 
+  if (!more && iter.empty())
+    return -ENODATA;
+
   return 0;
 }
 
+int rgw_usage_log_clear(cls_method_context_t hctx, bufferlist *in, bufferlist *out)
+{
+  CLS_LOG(10,"%s", __func__);
+  /* only continue if object exists! */
```

Comment:
why is this stat needed? will map_clear not do the right thing if the object doesn't exist?